package.json

```diff
@@ -29,9 +29,11 @@
     "browserify": "^12.0.1",
     "bundle-collapser": "^1.1.1",
     "coffee-script": "^1.8.0",
+    "core-js": "^2.2.1",
     "coveralls": "^2.11.6",
     "del": "^2.0.2",
     "derequire": "^2.0.3",
+    "es6-symbol": "^3.0.2",
```

Comment:
I have pushed a fix on your fork to remove the dependency, so that we can
unblock the code review. Le sam. 9 avr. 2016 à 00:56, Sam Balana <notifications@github.com> a écrit :
In package.json :

>      "coveralls": "^2.11.6",
>      "del": "^2.0.2",
>      "derequire": "^2.0.3",
> +    "es6-symbol": "^3.0.2",


Thank you @RaitoBezarius ! I forgot to remove this dependency when removing unit test support for
es6-symbol.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub



Kind regards, Ryan Lahfa